buildSrc/src/main/kotlin/com.hedera.block.tools.gradle.kts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("application")
+    id("com.hedera.block.conventions")
+    id("me.champeau.jmh")
+} //
+// val maven = publishing.publications.create<MavenPublication>("maven") { from(components["java"])
+// }
+//
+// signing.sign(maven)
```

Comment:
Can we remove these comments? If they're needed for an operation, can you document that?